### PR TITLE
Structure README.md

### PR DESCRIPTION
**Pull request by sypets**
- reorder the part about creating issues and add a numbered list for better readability
- add subheaders in README.md
- link to CONTRIBUTING.md
- Add information about how to upvote issues

### CODE REVIEW

**Comment by TPS** [comment]
@sypets Do you mind resolving the conflict I introduced by updating the GitHub contact link?

**Comment by sypets** [comment]
Sorry about the late reaction. Is fixed now.

File: README.md [hunk 1]
```diff
@@ -9,22 +11,39 @@ Issues and feature requests posted to GitHub are not stored in any
...
 If you are a GitHubber, and you would like to respond and close issues
 on this repository, please let me (or one of the other collaborators)
 know, and we'll add you as a collaborator.  Or maybe use some secret
 GitHub internal APIs to get access, if you have such things.  If you
 want to know what people are asking for, or ask for them to provide
 details about use cases etc, then head over to the issues list.
 
-Post an issue if you have an issue or feature request for GitHub.
+
+## If you have an issue or feature request for GitHub.
```
Comment (safinsingh) [inline README.md:24]: Extra period in heading

File: README.md [hunk 2]
```diff
@@ -9,22 +11,39 @@ Issues and feature requests posted to GitHub are not stored in any
...
+
+1. Search for existing [issues](https://github.com/isaacs/github/issues) here first
```
Comment (safinsingh) [inline README.md:26]: You can drop the `first` at the end, it's already the first item in an ordered list.

File: README.md [hunk 3]
```diff
@@ -9,22 +11,39 @@ Issues and feature requests posted to GitHub are not stored in any
...
+2. If you did not find any existing issue for your topic, post a new issue.
```
Comment (safinsingh) [inline README.md:27]: Extra period in single-sentence list item

**Review by safinsingh — CHANGES_REQUESTED** [review]

**Comment by sypets** [comment]
I cleaned up my workspace and deleted the fork because I did not realize I still had a PR open. I did not see any other possibility than to create a new PR: #1852 

I resolved all comments and added the changes. Sorry.

**Pull request closed without merging**